salt/wheel/file_roots.py

```diff
@@ -19,7 +19,7 @@ def find(path, saltenv="base"):
         return ret
     for root in __opts__["file_roots"][saltenv]:
         full = os.path.join(root, path)
-        if not salt.utils.verify.clean_path(root, full):
+        if not salt.utils.verify.clean_path(root, full, subdir=True):
```

Comment:
@dwoz I remember discussing this a while ago, we wanted to ensure we added some test coverage for some security scenarios. Which scenarios would you prefer tested here?

Reply:
Looks good to me.